views/admin/de/module_de_lang.php

```diff
@@ -165,15 +189,17 @@
     'wd_text_order_no_transactions' => 'Dieser Bestellung sind keine Transaktionen zugeordnet.',
     'wd_text_payment_action_pay' => 'Direktbuchung',
     'wd_text_payment_action_reserve' => 'Genehmigung',
+    'wd_text_quantity' => 'Stück',
     'wd_text_support' => 'Support',
     'wd_three_d_link_text' => 'Nicht-3-D-Secure- und 3-D-Secure-Limits',
+    'wd_timeStamp' => 'Datum',
     'wd_total_amount_not_in_range_text' => 'Gesamtsumme nicht innerhalb der erlaubten Bestellsumme für dieses Zahlungsmittel.',
     'wd_transactionID' => 'Transaction ID',
     'wd_transactionState' => 'Transaktionsstatus',
     'wd_transactionType' => 'Transaktionstyp',
     'wd_transaction_details_title' => 'Transaktionsdetails',
     'wd_transaction_response_details' => 'Antwort-Details',
-    'wd_trusted_shop_terms' => 'Mit der Übermittlung der für die Abwicklung des Rechnungskaufes und einer Identitäts- und Bonitätsprüfung erforderlichen Daten an payolution bin ich einverstanden. Meine <u><a href="%s" target="_blank">Einwilligung</a></u> kann ich jederzeit mit Wirkung für die Zukunft widerrufen.',
+    'wd_trusted_shop_terms' => 'Mit der Übermittlung der für die Abwicklung des Rechnungskaufes und einer Identitäts- und Bonitätsprüfung erforderlichen Daten an Payolution bin ich einverstanden. Meine <u><a href="%s" target="_blank">Einwilligung</a></u> kann ich jederzeit mit Wirkung für die Zukunft widerrufen.',
```

Comment:
Same here why is payolution written Capitalized, in the docu is always lowercase. Did you disscuss this with @hbrandl ?

Reply:
It was discussed in here: https://wd-cee.slack.com/archives/GFZKK7SCA/p1559573664024800?thread_ts=1559051638.019400&cid=GFZKK7SCA

Reply:
This key needs to be changed to 'wd_payolution_terms' disscussed with @hbrandl

Reply:
changed it in #132